TestCases/parallel_regression_AD.py

```diff
@@ -261,8 +261,8 @@ def main():
     discadj_fea           = TestCase('discadj_fea')
     discadj_fea.cfg_dir   = "disc_adj_fea"
     discadj_fea.cfg_file  = "configAD_fem.cfg" 
-    discadj_fea.test_iter = 9
-    discadj_fea.test_vals = [-6.070230, -6.262517, -0.000364, -8.708700] #last 4 columns
+    discadj_fea.test_iter = 4
```

Comment:
Why changing these test cases, are the results for iteration 9 still the same as before?

Reply:
This case is super sensitive to minor changes. I think that is because it is converged to machine precision, so as an experiment I'm converging it less...
The residuals at the 9th iteration were not the same (still in the -6 ballpark) but the derivatives were.